recipes/vectorclass/all/conanfile.py

```diff
@@ -14,13 +14,37 @@ class VectorclassConan(ConanFile):
     topics = ("conan", "vectorclass", "vector", "simd")
     homepage = "https://github.com/vectorclass/version2"
     url = "https://github.com/conan-io/conan-center-index"
-    no_copy_source = True
-    settings = "os", "arch", "compiler"
+    exports_sources = ["CMakeLists.txt"]
+    generators = "cmake"
+    settings = ("os", "arch", "compiler", "build_type")
+    options = {
+        "fPIC": [True, False],
+        "header_only": [True, False]
+    }
+    default_options = {
+        "fPIC": True,
+        "header_only": True
```

Comment:
I would remove header only and always compile this cpp file (without specific simd flags, I don't know if the pre compile binary is really useful).

Reply:
I also don't know whether having static library is useful for other people. Currently we use it for choosing between SSE2 and AVX2 implementations from isolated DLLs that also have different values for VCL_NAMESPACE.

Reply:
maybe it's better to just change default value to False?

Reply:
yes, I agree